src/ape/managers/chain.py

```diff
@@ -450,6 +458,66 @@ def get(
 
         return contract_type or default
 
+    def resolve_proxy(self, address: AddressType) -> Optional[AddressType]:
```

Comment:
We need to probably move this to `ape-ethereum` and the `EcosystemAPI`

`decode_proxy(address)`.

This way, plugins like `ape-starknet` can decode their own proxy standards.